pokemon-api/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/dto/TrainerAuthenticateDTO.java

```diff
@@ -0,0 +1,32 @@
+package cz.muni.fi.pa165.pokemon.league.participation.manager.dto;
+
+/**
+ * DTO class for autheticate Trainer.
+ *
+ * @author Jiří Medveď 38451
+ */
+public class TrainerAuthenticateDTO

```

Comment:
Missing equals, hashCode and toString methods, auto generated with all properties is completely enough.